lib/joi/joi.bundle.js

```diff
@@ -94,17 +94,20 @@ module.exports =
 
```

Comment:
Generated according to the instructions in https://github.com/OldSneerJaw/couchster/blob/v1.2.1/lib/joi/README.md